src/main/java/com/ibm/devops/connect/CloudPublisher.java

```diff
@@ -190,6 +317,38 @@ public static boolean uploadQualityData(HttpEntity entity) throws Exception {
         }
     }
 
+    public static void uploadQualityDataRaw(String payload) throws Exception {
```

Comment:
A lot of these seem like duplicate methods, with just a change here or there.  I wonder if there is a way that we could make one generic one that takes in certain input/strings that are different.

Reply:
Any suggestions?  Not seeing it after a quick look

Reply:
I guess these are all subtly different.  Maybe just do for builds and deploys as those look pretty similar?